### ISSUE TITLE
Fix Kubernetes daemonset example to include /host/etc

### ISSUE THREAD

**Issue by mfdii** [falcosecurity/falco#473]
In order for the eBPF probe loader to work properly on COS and minikube, /etc from the host needs to be mounted inside the Falco container.

The below needs added to the [daemonset](https://github.com/falcosecurity/falco/blob/dev/integrations/k8s-using-daemonset/k8s-with-rbac/falco-daemonset-configmap.yaml#L37):

```
            - mountPath: /host/usr
              name: usr-fs
              readOnly: true
            - mountPath: /host/etc
              name: etc-fs
              readOnly: true
            - mountPath: /etc/falco
              name: falco-config
...
        - name: usr-fs
          hostPath:
            path: /usr
        - name: etc-fs
          hostPath:
            path: /etc
        - name: falco-config
          configMap:
            name: falco-config
```

Opening this to remind myself to fix this.

**Comment by mfdii** [comment]
@nestorsalceda I am guessing the helm chart will need this fix as well.

**Comment by nestorsalceda** [comment]
In the helm chart the mountPath exists:

https://github.com/helm/charts/blob/master/stable/falco/templates/daemonset.yaml#L70

And is enabled by default:

https://github.com/helm/charts/blob/master/stable/falco/values.yaml#L57

So I think is working fine in the Chart, but thank you very much for the ping. I appreciate it :)

**Comment by mfdii** [comment]
Fixed in #475
